jigglypuffRL/deeprl/ddpg/ddpg.py

```diff
@@ -73,10 +81,15 @@ def __init__(
         self.max_ep_len = max_ep_len
         self.start_update = start_update
         self.update_interval = update_interval
+        self.save_interval = save_interval
         self.layers = layers
         self.tensorboard_log = tensorboard_log
         self.seed = seed
         self.render = render
+        self.network_type = network_type
+        self.pretrained = pretrained
+        self.save_name = save_name
+        self.save_version = save_version
```

Comment:
`Nit`
Over here could you also add statements for assigning the load/save function to a class variable.
Something like:
`self.save = save_params`
`self.load = load_params`

Reply:
I guess this just adds the functionality of having the common save and load functions as object methods